packages/fxa-settings/src/pages/SigninBounced/index.tsx

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps, Link, useNavigate } from '@reach/router';
+import { logPageViewEvent } from '../../lib/metrics';
+import { ReactComponent as EmailBounced } from './graphic_email_bounced.svg';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../models/hooks';
+
+export type SigninBouncedProps = {
+  onBackButtonClick?: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => void;
+  canGoBack?: boolean;
+  email: string;
+};
+
+const SigninBounced = ({
+  canGoBack,
+  email,
+  onBackButtonClick = () => window.history.back(),
+}: SigninBouncedProps & RouteComponentProps) => {
+  logPageViewEvent('signin-bounced', {
+    entrypoint_variation: 'react',
+  });
+  const ftlMessageResolver = useFtlMsgResolver();
+  const backText = ftlMessageResolver.getMsg('back', 'Back');
+  const navigate = useNavigate();
+
+  const createAccountHandler = () => {
+    // TO-DO (per content-server functionality):
+    // clear all existing accounts
+    // clear the session
+    // clear the form prefill
+    navigate('/signup');
+  };
+
+  // TO-DO:
+  // - fix up the ftl strings with comments and vars etc.
+  // - add in fade-in animations
+
+  return (
+    <>
+      <div className="mb-4">
+        <FtlMsg id="signin-bounced-header">
+          <h1 className="card-header">
+            Sorry. We’ve locked your{'\u00A0'}account.
+          </h1>
+        </FtlMsg>
+      </div>
+      <section>
+        <div className="flex justify-center mx-auto">
+          <EmailBounced className="w-3/5" role="img" />
+        </div>
+        <FtlMsg id="signin-bounced-message" vars={{ email: email }}>
+          <p className="text-sm mb-6">
+            The confirmation email we sent to {email} was returned and we’ve
+            locked your account to protect your Firefox data.
+          </p>
+        </FtlMsg>
+
+        <FtlMsg id="signin-bounced-help">
+          <p className="text-sm mb-6 text-grey-400">
+            If this is a valid email address,{' '}
+            <a
```

Comment:
Should this be a `LinkExternal` component from fxa-react instead of an anchor tag? It looks like there's a lot of overlap in the attributes

Reply:
oh yeah I'm sure it should!! I'll change it!

Reply:
fixed!